### PR TITLE
README update and more general cleanup.

### PR DESCRIPTION
**Pull request by aulb**
@reblws More cleanup needed.

### CODE REVIEW

File: src/helpers/utils.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import JOYO_KANJI from './constants';
+
+
+// Shuffle in place, but create a copy
+function shuffleJoyoKanji() {
+  // Duplicate our original list
+  const kanjiList = JOYO_KANJI.slice();
```
Comment (reblws) [inline src/helpers/utils.js]: use ES6 spreading `[...JOYO_KANJI]` instead of `.slice()`

File: src/helpers/utils.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+function getRandomColor() {
+  let letters = '0123456789ABCDEF';
+  let color = '#';
+  for (let i = 0; i < 6; i++) {
+      color += letters[Math.floor(Math.random() * 16)];
+  }
+  return color;
+}
+
+// Figure out some way to generate gradient
```
Comment (reblws) [inline src/helpers/utils.js:37]: we'll use this http://www.hsluv.org/

File: src/helpers/utils.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  let kanjiList = shuffleJoyoKanji().slice(0, numOfKanji);
+  kanjiList = kanjiList.map((character) => ({
+    character: character,
+    color: getRandomColor(),
+  }));
+
+  return kanjiList;
+}
+
+export default kanjiFactory;
```
Comment (reblws) [inline src/helpers/utils.js]: we're gonna want to make this file use named exports so remove this export default line

File: src/helpers/utils.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+// Figure out some way to generate gradient
+function getRandomInt(min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (Math.floor(max) - Math.ceil(min))) + Math.ceil(min);
+}
+
+// helper to generate test kanji until we connect to db
+function kanjiFactory(numOfKanji) {
```
Comment (reblws) [inline src/helpers/utils.js]: Make this a named export like `export function kanjiFactory(numOfKanji) { ... }`

then we can import any named exports like `import { NAME } from '../utils/helpers'`

File: src/helpers/utils.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  for (let i = 0; i < 6; i++) {
+      color += letters[Math.floor(Math.random() * 16)];
+  }
+  return color;
+}
+
+// Figure out some way to generate gradient
+function getRandomInt(min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
```
Comment (reblws) [inline src/helpers/utils.js]: I forgot to remove these lines 39 and 40 can you do that for me thanks

**Review by reblws — CHANGES_REQUESTED** [review]
Cool, just make utils.js export named functions and dont make anything default

File: src/helpers/utils.js (hunk 5 above)
Comment (aulb) [inline src/helpers/utils.js]: Why do we need to remove these?

File: src/helpers/utils.js (hunk 2 above)
Comment (aulb) [reply src/helpers/utils.js:37]: We'll do that later~

**Review by reblws — APPROVED** [review]
Excellent

**Pull request merged**